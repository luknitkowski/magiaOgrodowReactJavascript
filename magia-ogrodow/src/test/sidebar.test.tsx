import renderer from 'react-test-renderer'

import SideBar from '../components/sidebar';

test('SideBar renders correctly', () => {
  const sidebar = renderer.create(<SideBar />).toJSON();
  expect(sidebar).toMatchSnapshot();
});

