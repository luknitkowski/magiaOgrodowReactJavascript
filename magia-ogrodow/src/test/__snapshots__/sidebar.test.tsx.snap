// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideBar renders correctly 1`] = `
<div
  className="makeStyles-SideBarBlock-3"
>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-menuButton-4"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiButton-label"
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
        />
      </svg>
    </span>
  </button>
</div>
`;
